### PR TITLE
docs: add offerings availability label

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Fixes https://github.com/daytonaio/enterprise-docs/issues/173

### CODE REVIEW

**Comment by stefanicjuraj** [comment]
@ivan-burazin

File: src/content/docs/administration/license.mdx [hunk 1]
```diff
@@ -3,6 +3,12 @@ title: License
 description: Learn about the Daytona license and how to manage it.
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Cloud**, **Self-Managed**
```
Comment (ivan-burazin) [inline src/content/docs/administration/license.mdx:9]: @stefanicjuraj how is licence available in Cloud? This is just for Self managed as cloud users have "billing"

File: src/content/docs/administration/quotas.mdx [hunk 2]
```diff
@@ -3,6 +3,12 @@ title: Quotas
 description: Learn about the Daytona quotas and how to manage them.
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
```
Comment (ivan-burazin) [inline src/content/docs/administration/quotas.mdx:9]: @stefanicjuraj plus update the designs to the figma file. thanks

Comment (ivan-burazin) [reply src/content/docs/administration/quotas.mdx:9]: @stefanicjuraj the entire admin section is not available on Cloud or OSS

File: src/content/docs/installation/cluster.mdx [hunk 3]
```diff
@@ -5,6 +5,12 @@ sidebar:
   label: Cluster
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
```
Comment (ivan-burazin) [inline src/content/docs/installation/cluster.mdx:10]: @stefanicjuraj these three installation options are only visible to self manged

File: src/content/docs/misc/telemetry.mdx [hunk 4]
```diff
@@ -5,4 +5,10 @@ sidebar:
   label: Telemetry
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
```
Comment (ivan-burazin) [inline src/content/docs/misc/telemetry.mdx:11]: @stefanicjuraj telemetry is not available on cloud

File: src/content/docs/tools/jetbrains-gateway.mdx [hunk 5]
```diff
@@ -6,6 +6,11 @@ sidebar:
 ---
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
+import Label from '@components/Label.astro';
+
+<Label>
+  Distribution: **Cloud**, **Self-Managed**
```
Comment (ivan-burazin) [inline src/content/docs/tools/jetbrains-gateway.mdx:12]: @stefanicjuraj available on all three

**Review by ivan-burazin — CHANGES_REQUESTED** [review]

**Review by ivan-burazin — CHANGES_REQUESTED** [review]
- All Installation is only self Managed
- Usage is not OSS
- Settings is cloud and self managed
- CLI is all

**Comment by fabjanvucina** [comment]
@stefanicjuraj Before we merge, can you check if there are new pages (from recently merged PRs) that are missing the offerings label

**Review by fabjanvucina — APPROVED** [review]

**Pull request merged**